### PR TITLE
Remove old mock dependency

### PR DESCRIPTION
**Pull request by jiivan**
Since python3 `mock` is included in the standard library under `unittest.mock` and is no longer maintained as a separate package.

### CODE REVIEW

File: tests/apps/lux/task/test_luxverificator.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import os
-from mock import patch, Mock
+import unittest.mock as mock
+import unittest.mock as mock
```
Comment (Elfoniok) [inline tests/apps/lux/task/test_luxverificator.py]: ? Double declaration?

**Review by Elfoniok — COMMENTED** [review]
As a general note, wouldn't be easier to import Mock class? not mock namespace?

File: tests/apps/lux/task/test_luxverificator.py (hunk 1 above)
Comment (jiivan) [reply tests/apps/lux/task/test_luxverificator.py]: Fixed

**Comment by jiivan** [comment]
I like to avoid namespace pollution by importing modules instead of single classes/functions.

**Review by Elfoniok — APPROVED** [review]

**Pull request merged**